spec-0004/index.md

```diff
@@ -18,38 +19,99 @@ If relevant, include examples of how the new functionality would be used,
 
-We recommend that projects add a cron job to test against the nightly wheels and
-when errors are encountered to automatically open an issue in their project so they
-can investigate.
+### Build Nightly Wheels
 
-The nightly wheels can be found [here](https://anaconda.org/scientific-python-nightly-wheels/).
+There are a few steps to implementing this for your project:
 
-We have a GitHub action to upload new nightly wheels and remove old ones:
-https://github.com/scientific-python/upload-nightly-action
+1. Get access to https://anaconda.org/scientific-python-nightly-wheels/, the location used to host nightly wheels
+2. Setup a CI step that builds wheels for your project
+3. Setup a CI step that uploads wheels to https://anaconda.org/scientific-python-nightly-wheels/
 
-### Core Project Endorsement
+For step (1) contact nightly-wheels@scientific-python.org. Deposit the token you get as a [secret on your repository](https://docs.github.com/en/actions/security-guides/encrypted-secrets) named `UPLOAD_TOKEN`.
+
+The work for step (2) depends on your project. You are probably already doing this for your
+releases. The new thing to add is that building wheels is run on a schedule every night or
+once a week.
+
+For step (3) there is a GitHub Action that you can use. You can find the action at
+https://github.com/scientific-python/upload-nightly-action. To use it in your "build wheels
+workflow" add the following lines as an additional step:
+
+```
+- name: Upload wheel
+  uses: scientific-python/upload-nightly-action@main
+  with:
+    artifacts_path: dist/
+    anaconda_nightly_upload_token: ${{ secrets.UPLOAD_TOKEN }}
+```
+
+Complete examples of how projects implement this in their CI setup are linked in the Notes section.
+
+#### Process for Adding New Projects
+
+After someone emails nightly-wheels@scientific-python.org requesting access to upload wheels
+a human has to respond to that request.
+
+We want to be open to projects uploading wheels but at the same time need to perform some
+amount of due dilligence before giving people access. This is because once a user is given
+access they could upload wheels for any project. We assume that people are not malicious
+and we can see from the logs who misbehaved after the fact.
+
+Once you have established who the person is that contacted nightly-wheels@scientific-python.org
+and that they represent the project they say they are representing reply to the email.
+Ask the person to create an account on https://anaconda.org and tell you the username.
+You can then add them to the `scientific-python-nightly-wheels` organisation on anaconda.org.
+Let the user know that they have been added and that they can create a access token at
+https://anaconda.org/scientific-python-nightly-wheels/settings/access. The token should
+only have the "Allow uploads to Standard Python repositories" scope and use the project
+name as the token name.
```

Comment:
I guess if we add them to the org, we can just ask them to open an issue, mail communicaiton is necessary only if we create the token for them...

Reply:
Good point. Should we use https://github.com/scientific-python/upload-nightly-action as the place to create an issue in?

Reply:
For the current location @ogrisel said he added additional owners when he created it and those also helped add other owners. Based on trust and trust in those you trust. I think that is a good enough model for what we are doing here (nightlies). Might be nice to leave a comment/issue somewhere when you add a person to create a paper trail of who added who, but /shrugs.

Reply:
Implemented a change to use a GitHub repo instead of email.